ingestion/src/metadata/cli/backup.py

```diff
@@ -132,7 +133,7 @@ def upload_backup_azure(account_url: str, container: str, file: Path) -> None:
         )
         raise err
 
-    click.secho(
+    print(
```

Comment:
I would use `print_ansi_encoded_string` for all the prints, so if we want to change how we print the messages in the future, we have to update only the `print_ansi_encoded_string` method. WDYT?

Reply:
makes sense, let me make that change